running/architecture.rst

```diff
@@ -2,3 +2,72 @@ Architecture
 ============
 
 Deconst is distributed as a set of Docker containers, deployed by an Ansible playbook.
+
+Terminology
+-----------
+
+.. glossary::
+
+  content repository
+    Version controlled repository containing markup that provides content for some subset of the
+    completed site. To begin, we'll assume that all of these are in ``git`` repositories on
+    `GitHub <github.com>`_. Most of our documentation will be in Sphinx reStructuredText, but the
+    developer blog will be in Markdown as part of a Jekyll site.
+
+  map repository
+    Version controlled repository containing plain-text documents that associate subtrees of
+    indexed content, identified by a :term:`content ID` prefix, with subtrees of :term:`paths`
+    on the presented site.
+
+  content ID
+    Unique identifier assigned to a single page of content. By convention, these are URLs that
+    join the https URL of the GitHub repository, ``tree/branch`` if "branch" is not master, and
+    the subpath of the page within the *rendered* output of the documentation build tool. Most of
+    the architecture should treat these as opaque strings. Examples:
+    this page would be ``https://github.com/deconst/deconst-docs/running/architecture`` on master,
+    or ``https://github.com/deconst/deconst-docs/tree/glossary/running/architecture`` until this
+    pull request is merged.
+
+  path
+  paths
+    Subpath within the final presented content of a deconst site. Includes the fully-qualified
+    domain name of the page. For example:
+    ``developer.rackspace.com/sdks/cloud-servers/getting-started/``.
+
+  layout
+    Template of common markup that surrounds each presented page with navigation, brand identity,
+    copyright information and anything else that's shared among some subset of each site.
+
+Components
+----------
+
+.. glossary::
+
+  normalizer
```

Comment:
This is what I came up with to replace "renderer", by the way. If the rest of you like it I'll rename the repository accordingly.

I think it's a little better, because its major task is normalizing content to meet our JSON schema, but it does have additional responsibilities that aren't captured by the name: namely, local presentation for pull requests and content submission for merged ones.